docs/migration/css-changes-in-version-5.md

```diff
@@ -20,3 +20,7 @@ title: Changes to Styles in 5.0
 ## Change in Configurator Required Error Message Mixin
 
 - `cx-configurator-error-msg` mixin has been defined on `%cx-configurator-required-error-msg` mixin to achieve a consistent styling for rendering an error and validation messages.
+
+## Change in Breadcrumb component style structure 
```

Comment:
```suggestion
## Changes in styles

### Breadcrumb component 
```

Reply:
I think it's better we create separate section to gather all style changes. There might be more coming.